lib/kryptonite/aes.ex

```diff
@@ -244,4 +346,9 @@ defmodule Kryptonite.AES do
 
   @spec cut_key(binary) :: binary
   defp cut_key(<<key::binary-size(@key_byte_size), _::binary>>), do: key
+
+  defp do_stream_encrypt(elem, acc) do
```

Comment:
Maybe we could have this a `def` instead of `defp` so later we could unit-test it in isolation?